platforms/software/build-init/src/main/java/org/gradle/buildinit/plugins/internal/GitIgnoreGenerator.java

```diff
@@ -68,15 +68,18 @@ private static Set<String> getGitignoresToAppend(File gitignoreFile) {
         return result;
     }
 
-    private static List<String> withComment(String entry) {
+    private static List<String> withComment(InitSettings settings, String entry) {
         List<String> result = new ArrayList<>();
-        if (entry.startsWith(".gradle")) {
-            result.add("# Ignore Gradle project-specific cache directory");
-        } else if (entry.startsWith("build")) {
-            result.add("# Ignore Gradle build output directory");
+
+        if (settings.isWithComments()) {
+            if (entry.startsWith(".gradle")) {
+                result.add("# Ignore Gradle project-specific cache directory");
```

Comment:
🤔 Same comment as for `.gitattributes`

Reply:
Reverted changes for comments in git-related files